Formula/libui.rb

```diff
@@ -0,0 +1,50 @@
+class Libui < Formula
+  desc "portable GUI library for C"
+  homepage "https://github.com/andlabs/libui"
+  url "https://github.com/andlabs/libui/archive/alpha3.1.tar.gz"
```

Comment:
Despite being v3, ~2.8k commits, 4k stars, this library doesn't have a version tagged `stable` yet. Is there an alternate tap this could be submitted to?